roles/edpm_nova/defaults/main.yml

```diff
@@ -32,14 +32,14 @@ edpm_nova_compute_image: "quay.io/podified-antelope-centos9/openstack-nova-compu
 # certs
 edpm_nova_tls_certs_enabled: "{{ edpm_tls_certs_enabled | default(False) }}"
 
-# list of services to stop during adoption
-edpm_old_tripleo_compute_sevices:
+# list of tripleo nova services to stop during EDPM adoption
+edpm_nova_old_tripleo_compute_sevices:
   - tripleo_nova_compute.service
-  - tripleo_nova_libvirt.target
   - tripleo_nova_migration_target.service
-  - tripleo_nova_virtlogd_wrapper.service
-  - tripleo_nova_virtnodedevd.service
-  - tripleo_nova_virtproxyd.service
-  - tripleo_nova_virtqemud.service
-  - tripleo_nova_virtsecretd.service
-  - tripleo_nova_virtstoraged.service
+  # Other control plane services, in case of all-in-one overcloud adoption
```

Comment:
hum ok yes if you were adopting form standalone this would be needed.

Reply:
there is also all-in-one overcloud managed with Director supported in OSP (which is NOT standalone tripleo)